apps/examples/testcase/tc_main.c

```diff
@@ -106,10 +110,10 @@ extern int utc_mqtt_main(int argc, char *argv[]);
 extern int itc_mqtt_main(int argc, char *argv[]);
 extern int utc_sysio_main(int argc, char *argv[]);
 extern int itc_sysio_main(int argc, char *argv[]);
-extern int utc_task_manager_main(int argc, char *argv[]);
-extern int itc_task_manager_main(int argc, char *argv[]);
-extern int utc_wifi_manager_main(int argc, char *argv[]);
-extern int itc_wifi_manager_main(int argc, char *argv[]);
+extern int utc_taskmanager_main(int argc, char *argv[]);
```

Comment:
it's already updated at the latest git.
please check it.

Reply:
ok